### PR TITLE
Tweak readme, add `CONTRIBUTING.md` and `CONTRIBUTORS.md`

### PR DESCRIPTION
**Pull request by Steveplays28**
Updated the roadmap, installation (now development) section, contributing, contact, and uncentered the text below the banner. Also removed the back to top links, added `CONTRIBUTING.md` and `CONTRIBUTORS.md`, and fixed some typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,116 +59,108 @@ Key features:
 * Enchanting support
 * Support for adding and customizing materials, patterns and gems entirely trough config files
```
Comment (SigmundGranaas) [inline README.md]: Patterns was the old name for schematics, so I think we should replace the use of pattern with schematics

File: README.md [hunk 2]
```diff
@@ -88,116 +59,108 @@ Key features:
...
 
+<!-- ROADMAP -->
 
-<!-- GETTING STARTED -->
+## Roadmap
 
-## Getting Started with modding
+Forgero is in active development, and features are being added frequently.  
+We support [Semantic Versioning](https://semver.org/), meaning we make sure we break your saves as little as possible
+with updates, and if possible only in major releases.
```
Comment (SigmundGranaas) [inline README.md]: We support [Semantic Versioning](https://semver.org/), which means we make sure to not introduce breaking changes unless we are moving to a new major version.

**Review by SigmundGranaas — APPROVED** [review]
Looks really solid! Just two small comments. Nice work!

File: README.md (hunk 2 above)
Comment (Steveplays28) [reply README.md]: That is a much better way to word it, thanks!

**Pull request merged**